src/main/java/lotto/LottoApplication.java

```diff
@@ -0,0 +1,24 @@
+package lotto;
+
+import lotto.controller.LottoMachine;
+import lotto.controller.WinningLotto;
+import lotto.domain.Lotto;
+import lotto.ui.InputView;
+import lotto.ui.ResultView;
+import lotto.util.LottoNumberParser;
+
+public class LottoApplication {
+    public static void main(String[] args) {
+        String purchaseAmount = InputView.getPurchaseAmount();
+        LottoMachine lottoMachine = new LottoMachine(purchaseAmount);
+        Lotto lotto = lottoMachine.generate();
+        ResultView.printLottoCount(lotto.count());
+        ResultView.printLottoNumbersList(lotto.toString());
```

Comment:
```suggestion
        ResultView.printLotto(lotto);
```
위와 같이 구현하는 것도 한 방법일 듯